report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,50 +1,50 @@
-\section{Problem Definition}
+\section{Problem}
 \label{sec:problem_definition}
 
-Our work aims to improve the accuracy and robustness of major oxide predictions derived from \gls{libs} data, building upon the baseline established in \citet{p9_paper}.
-There are many challenges in predicting major oxides from \gls{libs} data, including the high dimensionality and non-linearity of the data, as well as the presence of multicollinearity.
-Some of these are caused by \textit{matrix effects}\cite{andersonImprovedAccuracyQuantitative2017}, which is a catch-all term for any effect that can cause the intensity of emission lines from an element to vary, independent of that element's concentration. So it's unknown variables that affect the results.
+Predicting major oxide compositions from \gls{libs} data presents significant computational challenges, including the high dimensionality and non-linearity of the data, compounded by multicollinearity and the phenomenon known as \textit{matrix effects}.
+These effects can cause the intensity of emission lines from an element to vary independently of that element's concentration, introducing unknown variables that complicate the analysis.
 Furthermore, due to the high cost of data collection, datasets are often small, which further complicates the task of building accurate and robust models.
 
-Based on the limitations with the current \gls{moc} pipeline, as reported in \citet{p9_paper}, we identified three key areas for further investigation: dimensionality reduction, model selection, and outlier removal.
+Our work, building upon the baseline established in \citet{p9_paper}, aims to address the significant challenges inherent in predicting major oxide compositions from \gls{libs} data by improving the accuracy and robustness of these predictions.
```

Comment:
```suggestion
Building upon the baseline established in \citet{p9_paper}, our work aims to address the significant challenges inherent in predicting major oxide compositions from \gls{libs} data by improving the accuracy and robustness of these predictions.
```
Original phrasing was fine but two interposed sentences in a row felt a bit difficult to read, and this one was the easiest to change rather than the sentence directly following.

Reply:
I actually changed it away from that because I wanted `our work` to lead the sentence. Here's the original I came up with:

```suggestion
To build upon the baseline established in \citet{p9_paper}, our work aims to address these challenges by improving the accuracy and robustness of major oxide predictions derived from \gls{libs} data.
```

Thoughts?

Reply:
Either works - could also change the next sentence and remove the interposed sentence there instead. I just didn't want two in a row.